pkgs/tools/security/nitrokey-app/cppcodec.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
As this seems to be latest release, it can be moved to development/libraries and introduce in a separate commit.
We can vendor it again if needed.